spec/models/note_spec.rb

```diff
@@ -1,5 +1,231 @@
 require 'rails_helper'
 
-RSpec.describe Note, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+shared_examples 'a valid note' do
+  it 'and passes validation' do
```

Comment:
Recordá que los its deberían poder leerse como oraciones por si solas, o anexadas a los contextos superiores
```suggestion
  it 'passes the content_length validation' do
```
Una forma de testear esto es hacer fallar el test y verificar como se muestra en la consola